packages/reference/src/components/LinkActions/CombinedLinkActions.tsx

```diff
@@ -97,32 +96,34 @@ function CombinedAssetLinkActions(props: LinkActionsProps) {
   if (!props.canLinkEntity || !props.canCreateEntity) {
     if (props.canLinkEntity) {
       return (
-        <TextLink
+        <Button
```

Comment:
I personally like that the existing one has primary color, so reusing the content is prioritized, but was it approved by design?

Reply:
Yes. I personally think it looks out of place in the editor when the whole button is primary.

Reply:
"when the whole button is primary" - what do you mean here?

Reply:
With the button the primary colour is quite overpowering and draws your attention since the buttons are so big. Even with `primary` I don't think `TextLink` draws your attention that much, so while `primary` works well for `TextLink` I don't think that's the case with buttons here.

Reply:
so, should we use muted for both of them maybe?